packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -248,6 +255,21 @@ class SendConfirmation extends React.Component<Props, State> {
       </View>
     )
   }
+
+  render() {
+    const { account } = this.props
+    if (!account) {
+      throw Error('Account is required')
```

Comment:
Similar question on error handling - when will we ever reach this state?

Reply:
This could never happen to our users (unless we mess something and `SendConfirmation` is displayed before an account is created 😸)

Do you have another way to handle this?

Reply:
We generally don't do much invariant checking in our render functions. And account is something that we can generally rely on being set up properly. I'd recommend just removing this check

Reply:
We can't really remove it otherwise TS complains about `account` which can be null.